### PR TITLE
[WIP] Get the serverless stack working and deployable

### PR DESCRIPTION
**Pull request by caggle**
The goal of this PR is to get a the serverless stack working, and then get it to deploy with our pipeline.

### CODE REVIEW

**Comment by caggle** [comment]
As of https://github.com/mozilla/mozdef-event-framework/pull/7/commits/9f86f45aff52e6c73904e9b0d9831a8ea7c068b3, this serverless stack successfully deploys and works, _manually_. (i.e. from my command line via `sls deploy ...`.

Next step is to try to get this deployed with our pipeline.

**Comment by caggle** [comment]
With https://github.com/mozilla/mozdef-event-framework/pull/7/commits/01f937f491f5a24a97a4eff5a116a89105441c50 this Serverless stack now deploys.

@Phrozyn I think this is ready for review. There is still work to be done in https://github.com/mozilla/mozdef-event-framework/pull/5 though.

File: functions/handler.py
Comment (Phrozyn) [inline functions/handler.py]: I don't think aws_event accurately describes this, api_event might be better suited.

File: functions/handler.py
Comment (Phrozyn) [inline functions/handler.py:37]: Curious why we removed this, it only renames specific pieces of the event to the format ES can ingest. It does not restrict the sending to sqs if these don't exist.

**Review by Phrozyn — COMMENTED** [review]
Overall, I love it, but had some questions, so I'll wait till we can talk before I approve.

File: functions/handler.py
Comment (caggle) [reply functions/handler.py]: Sure, I can change it to that.

Just to make sure, the `aws_event` or `api_event` we are referring here is going to be something like this: https://serverless.com/framework/docs/providers/aws/events/apigateway#example-lambda-proxy-event-default and it's different than the external events we would like to consume. The external events (stuff that's submitted to this endpoint) is in the `event` key of that JSON/dictionary.

File: functions/handler.py
Comment (Phrozyn) [reply functions/handler.py:37]: I think we should still detect if a payload is present, all webhooks should have this  field somewhere in the json. Unless you think this handler could be used for much more than just webhooks, curious as to your thoughts?

File: functions/handler.py
Comment (Phrozyn) [reply functions/handler.py]: It's not an aws event was all I was trying to point out, it's an api event, regardless of all the fluff the api-gateway tacks onto the accept headers. Everything in the RequestBody is what we are dealing with and interested in, not much else outside of that other than making sure it is sourced from the place we expect it to be, and authenticated properly, which should be vallidated by the authentication piece before it ever makes it this far.

File: functions/handler.py
Comment (caggle) [reply functions/handler.py:37]: There is a point in saying the handler could be used for taking events other than from webhooks, but if we have to do that we can make a different handler / function / endpoint to address that I think.

I will add checking the presence of a payload as well.

**Comment by caggle** [comment]
@Phrozyn if everything looks good I think this is ready for final review & merge.

**Review by Phrozyn — APPROVED** [review]

**Pull request merged**
